stablehlo/reference/Tensor.h

```diff
@@ -46,6 +46,9 @@ class Buffer : public llvm::RefCountedBase<Buffer> {
   /// Returns type of the Buffer object.
   ShapedType getType() { return type_; }
 
+  /// Updates type of the Buffer object.
```

Comment:
Any specific reason on the use of "///" instead of "//"?

Reply:
As per https://llvm.org/docs/CodingStandards.html#comment-formatting, I am trying to follow doxygen documentation comments for classes.